app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError
+                        )
```

Comment:
fix the import according to the best practicies

Reply:
here is in app.errors module we don't have too many classes to be imported... so, I would like to use * 
from app.errors import *

please let me know if in this case it will be exactable

Reply:
It is better not to use * in imports